bindgen-tests/tests/expectations/tests/generated/wrap_static_fns.c

```diff
@@ -11,3 +11,21 @@ int takes_alias__extern(func f) { return takes_alias(f); }
 int takes_qualified__extern(const int *const *arg) { return takes_qualified(arg); }
 enum foo takes_enum__extern(const enum foo f) { return takes_enum(f); }
 void nevermore__extern(void) { nevermore(); }
+void no_extra_argument__extern(__builtin_va_list va) { no_extra_argument(va); }
```

Comment:
this would end up needing `#include <stdarg.h>` wouldn't it?

I think you can modify the code (maybe add a field to `BindgenContext` so we can know if at least one variadic function was found and then add the `#include` if that's the case.

Reply:
Only in this particular case. In the general case people would be using `va_list` and not `__builtin_va_list` and because `va_list`, `va_start`, ... are defined in `#include <stdarg.h>` there would be no need to include it manually as it would already be included.

Would you want me to include it anyway ?

Reply:
well yeah that makes sense. At the end of the day if all input headers are included it also means that `stdarg` will be included too.